Makefile

```diff
@@ -42,3 +42,6 @@ test-client:
 # To run a specific test: TEST=<test name> make test-server
 test-server:
 	FLASK_ENV=test python3 -m pipenv run python -m pytest -k '${TEST}' --ignore=arlo-client
+
+test-server-coverage:
+	python3 -m pipenv run python -m pytest --cov=arlo_server/
```

Comment:
Do we need `FLASK_ENV=test`? what about `--ignore=arlo-client`? Do we want `-k '${TEST}'`?

Reply:
I don't think we should do coverage with paring down to one test, but yeah I think we need to tweak some other settings before we merge. I should have marked this as draft. Not planning on merging until we at least figure out how to cover everything properly.